File: python/prism_sdk/throughput_knowledge_representation_research_copilot.py
"""Worldgen P04 AFA-worldgen-P04-F11 product wrapper."""
from .worldgen_knowledge_copilot_support import KnowledgeCopilotRequest, KnowledgeCopilotReceipt, manifest as _manifest, run as _run
FEATURE_ID="AFA-worldgen-P04-F11"
CONTRACT_VERSION="worldgen-throughput-knowledge-copilot/1.0"
def worldgen_throughput_knowledge_representation_research_copilot_manifest(): return _manifest(feature_id=FEATURE_ID, contract_version=CONTRACT_VERSION, input_schema="KnowledgeCopilotRequest1@1", scale="prospective high-throughput", autonomy_tier="A2")
def run_worldgen_throughput_knowledge_representation_copilot(request): return _run(request, feature_id=FEATURE_ID, contract_version=CONTRACT_VERSION, scale="prospective high-throughput", require_approval=True, require_federation=False)
__all__=["KnowledgeCopilotRequest","KnowledgeCopilotReceipt","worldgen_throughput_knowledge_representation_research_copilot_manifest","run_worldgen_throughput_knowledge_representation_copilot"]

File: crates/worldgen/src/throughput_knowledge_representation_research_copilot.rs
//! Worldgen P04-F11 prospective high-throughput knowledge-representation research copilot.
use super::knowledge_copilot_support::{self, KnowledgeCopilotRequest, KnowledgeCopilotReceipt};
pub const FEATURE_ID:&str="AFA-worldgen-P04-F11"; pub const CONTRACT_VERSION:&str="worldgen-throughput-knowledge-copilot/1.0";
pub fn worldgen_throughput_knowledge_representation_research_copilot_manifest()->serde_json::Value{knowledge_copilot_support::manifest(FEATURE_ID,CONTRACT_VERSION,"KnowledgeCopilotRequest1@1","prospective high-throughput","A2")}
pub fn run_worldgen_throughput_knowledge_representation_research_copilot(request:&KnowledgeCopilotRequest)->Result<KnowledgeCopilotReceipt,knowledge_copilot_support::KnowledgeCopilotError>{knowledge_copilot_support::run(request,FEATURE_ID,CONTRACT_VERSION,"prospective high-throughput",true,true)}
pub use knowledge_copilot_support::{KnowledgeCopilotError,KnowledgeCopilotReceipt as WorldgenThroughputKnowledgeCopilotReceipt,KnowledgeCopilotRequest as WorldgenThroughputKnowledgeCopilotRequest};
